### ISSUE TITLE
Accessibility improvements

### ISSUE THREAD

**Issue by adamwoodnz** [WordPress/wporg-two-factor#61]
Issues found:

1. No focus styling on profile page Account action buttons
2. Password page 'show password' button has no accessible name
![Screen Shot 2023-02-15 at 10 55 14 AM](https://user-images.githubusercontent.com/1017872/218872148-54227fe7-efc7-4395-999c-017af78450cf.jpg)
3. No focus styling on 2FA page QR code or cancel button 
4. 2FA page QR code has no discernible text
![Screen Shot 2023-02-15 at 10 50 23 AM](https://user-images.githubusercontent.com/1017872/218871151-081b026b-c0b8-47ec-afa2-bedd1feb77c5.jpg)
5. Success notifications when focused say 'Dismiss this notice, button' but they aren't dismissable via mouse or keyboard
![Screen Shot 2023-02-15 at 11 17 31 AM](https://user-images.githubusercontent.com/1017872/218875896-d0f9fe3c-0ae7-490b-8ae4-6718cc6ace79.jpg)

Questions

1. Should we add UI which is keyboard accessible for easy copy and paste of backup codes?
2. Should the 'password best practices' link open in a new tab? Feels disruptive to navigate directly to an info link.

**Comment by iandunn** [comment]
> Success notifications when focused say 'Dismiss this notice, button' but they aren't dismissable

🤔 , IMO they shouldn't be dismissable, so the fix would be removing the label. I'd assume `isDismissible=true` param for the `SnackBar` component should do that.

> Should we add UI which is keyboard accessible for easy copy and paste of backup codes?

Do you mean adding new UI elements? What would they look like/do?


> Should the 'password best practices' link open in a new tab? Feels disruptive to navigate directly to an info link.

Usually I'd say it's best to let the user decide, but this may be an exception. I'd lean towards keeping it how it is, but don't feel strongly

https://www.nngroup.com/articles/new-browser-windows-and-tabs/

**Comment by adamwoodnz** [comment]
> Do you mean adding new UI elements? What would they look like/do?

I was thinking something like

![image](https://user-images.githubusercontent.com/1017872/218892298-2fa459db-a722-4a87-9d97-7c0283576e25.png)

Depends if we're really pushing users to print. In that case perhaps a print button might be helpful.

**Comment by adamwoodnz** [comment]
> Usually I'd say it's best to let the user decide, but this may be an exception. I'd lean towards keeping it how it is, but don't feel strongly
> 
> https://www.nngroup.com/articles/new-browser-windows-and-tabs/

Good read, thanks. Yeah I can't say this info is reference material that should be left open, so you're probably right.

**Comment by iandunn** [comment]
Ah, yes, those UI elements are planned in #45 . Feel free to pick that one up if you're interested 👍🏻
